README.md

```diff
@@ -1,50 +1,43 @@
 # Universal Permissionless Rewards Distributor
 
-A universal permissionless rewards distributor written in Solidity. It allows the distribution of any reward token (different reward tokens are possible simultaneously) based on a Merkle tree distribution, and using ERC20 allowance of a treasury.
+A universal permissionless rewards distributor written in Solidity.
+It allows the distribution of any reward token (different reward tokens are possible simultaneously) based on a Merkle tree distribution, and using ERC20 allowance of a treasury.
 
-The singleton contract allows any treasury to distribute rewards to any address, based on a Merkle tree. The Merkle root is stored in the contract. The Merkle root can be updated by whitelisted users. The distribution owner can freeze, force update, or suggest a new treasury. The treasury must accept the role and set an allowance to distribute rewards.
+The singleton contract allows any treasury to distribute rewards to any address, based on a Merkle tree.
+The Merkle root is stored in the contract.
+The Merkle root can be updated by whitelisted users.
+The distribution owner can freeze, force update, or suggest a new treasury.
+The treasury must accept the role and set an allowance to distribute rewards.
 
 Based on [Morpho's rewards distributor](https://github.com/morpho-dao/morpho-v1/blob/main/src/common/rewards-distribution/RewardsDistributor.sol), itself based on [Euler's rewards distributor](https://github.com/euler-xyz/euler-contracts/blob/master/contracts/mining/EulDistributor.sol).
 
 Tests are using [Murky](https://github.com/dmfxyz/murky), to generate Merkle trees in Solidity.
 
 ## Usage
 
-Merkle trees should be generated with [Openzeppelin library](https://github.com/OpenZeppelin/merkle-tree).
-It will ensure that trees will be secure for on-chain verification.
-
-## Installation
-
-Download foundry:
+Running tests requires `forge` from [Foundry](https://book.getfoundry.sh/getting-started/installation).
```

Comment:
I don't think we should be explaining how to install foundry, but rather point to how to where it's explained (so that we don't have to maintain it, and it is more up-to-date)